vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -151,6 +151,10 @@ func main() {
 		as.Serve(w, r)
 		healthCheck.UpdateLastActivity()
 	})
+	http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
+		healthCheck.ServeHTTP(w, r)
```

Comment:
could just respond with 200, but we were using this activity thing so assumed we should just use it here

Reply:
Not as far as I can tell. Why? Have you seen evidence of that happening?

Reply:
Oh, I see this: https://github.com/kubernetes/autoscaler/blob/1d5f0471bce0ad7183459c435667f3551be7d0d7/vertical-pod-autoscaler/pkg/utils/metrics/healthcheck.go